addons/medical_gui/functions/fnc_updateActions.sqf

```diff
@@ -58,8 +58,17 @@ private _shownIndex = 0;
             private _counts = [_items] call FUNC(countTreatmentItems);
             _countText = _counts call FUNC(formatItemCounts);
         };
+        _ctrl ctrlSetTooltipColorText [1, 1, 1, 1];
         _ctrl ctrlSetTooltip _countText;
 
+        // Show warning if tourniquet will interfere with action
+        if (GVAR(tourniquetWarning)
+        && {(_category in ["examine", "medication"]) || (_items findIf {"IV" in _x}) > -1}
```

Comment:
```suggestion
        && {(_category in ["examine", "medication"]) || (_items findIf {"IV" in _x}) > -1}
```

Still don't like the IV check, but anything else requires config lookups/caching items.

Reply:
I don't really have the means to properly investigate the solution I am proposing, but maybe if https://github.com/acemod/ACE3/blob/46473601d60a674f9d6c8ff8d02b1b535d1aea01/addons/medical_gui/functions/fnc_updateActions.sqf#L41 gives the statement defined e.g. here https://github.com/acemod/ACE3/blob/f774eaf787f32670926259f470abd5b0303d0357/addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp#L157 we could use that instead?

Reply:
It's still specific, but the medical system wasn't designed with this in mind (and I don't see a good way to implement it). It's fine. I think most 3rd party IVs will have "IV" in their classname.

Reply:
If anything, enforcing the " IV" (notice the leading space) name requirement would be 1 way to ensure support with ACE outside config, assuming the 3rd-party dev gives a fuck.